components/clickup/actions/create-chat-view-comment/create-chat-view-comment.mjs

```diff
@@ -0,0 +1,102 @@
+import clickup from "../../clickup.app.mjs";
+
+export default {
```

Comment:
If you put `common.mjs` in the `actions/common` folder, you would include it like this. This applies to all action components.
```suggestion
import clickup from "../../clickup.app.mjs";
import common from "../common/common.mjs";

export default {
  ...common,
```